ReactiveCocoa/Swift/Scheduler.swift

```diff
@@ -261,20 +292,22 @@ public final class TestScheduler: DateSchedulerType {
 	///
 	/// Optionally returns a disposable that can be used to cancel the work
 	/// before it begins.
-	public func scheduleAfter(interval: NSTimeInterval, action: () -> Void) -> Disposable? {
-		return scheduleAfter(currentDate.dateByAddingTimeInterval(interval), action: action)
+	@discardableResult
+	public func schedule(delay: TimeInterval, action: () -> Void) -> Disposable? {
```

Comment:
I think this should be `schedule(after delay: TimeInterval, action: () -> Void) -> Disposable?`